### PR TITLE
3d volume notebook

### PR DESCRIPTION
**Pull request by emmanuelle**
Doc upgrade checklist:

- [x] random seed is set if using random data
- [x] file has been moved from `unconverted/x/y.md` to `x/y.md`
- [x] old boilerplate at top and bottom of file has been removed
- [x] Every example is independently runnable and is optimized for short line count
- [x] no more `plot()` or `iplot()`
- [x] `graph_objs` has been renamed to `graph_objects`
- [x] `fig = <something>` call is high up in each example
- [x] minimal creation of intermediate `trace` objects
- [x] liberal use of `add_trace` and `update_layout`
- [x] `fig.show()` at the end of each example
- [ ] `px` example at the top if appropriate
- [x] minimize usage of hex codes for colors in favour of those in https://github.com/plotly/plotly.py-docs/issues/14

Closes https://github.com/plotly/documentation/issues/1492

This is a new notebook.

### CODE REVIEW

**Comment by emmanuelle** [comment]
I still have to generate the thumbnail, but I'm pushing first the tutorial so that you can comment

**Comment by emmanuelle** [comment]
The thumbnail is online https://plotly-tutorials.s3-us-west-1.amazonaws.com/plotly-documentation/thumbnail/3d-volume-plots.jpg

File: python/3d-volume.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+    pygments_lexer: ipython3
+    version: 3.7.3
+  plotly:
+    description: How to make 3D Volume Plots in Python with Plotly.
+    display_as: 3d_charts
+    has_thumbnail: true
+    language: python
+    layout: user-guide
+    name: 3D Volume Plots
+    order: 12.1.1
```
Comment (nicolaskruchten) [inline python/3d-volume.md]: this isn't a valid number :)

**Comment by nicolaskruchten** [comment]
Hmm so my laptop is really struggling to render these... are these plots particularly complex or...?

**Comment by nicolaskruchten** [comment]
yeah my Chrome tab crashed trying to render :(

**Comment by emmanuelle** [comment]
Yes these plots are quite resource-intensive to render... which surprised me since the arrays are not that big. I can try to decrease a bit the size of arrays but it'd interesting to see if @archmoj thinks this sluggishness is normal or not...

**Comment by emmanuelle** [comment]
I reduced the grid size it should behave better now.

**Comment by nicolaskruchten** [comment]
Works for me! 💃

**Pull request merged**